projects/github-actions/repo-gardening/src/tasks/triage-issues/index.js

```diff
@@ -11,6 +14,112 @@ const sendSlackMessage = require( '../../utils/slack/send-slack-message' );
 
 /* global GitHub, WebhookPayloadIssue */
 
+/**
+ * Request a list of matching labels from Open AI that can be applied to the issue,
+ * based on the issue contents.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} title   - Issue title.
+ * @param {string} body    - Issue body.
+ *
+ * @return {Promise<Object>} Promise resolving to an object of labels to apply to the issue, and their explanations.
+ */
+async function fetchOpenAiLabelsSuggestions( octokit, owner, repo, title, body ) {
+	const suggestions = { labels: [], explanations: {} };
+
+	// Get all the Feature and Feature Group labels in the repo.
+	const pattern = /^(\[Feature\]|\[Feature Group\])/;
+	const repoLabels = await getAllLabels( octokit, owner, repo, pattern );
+
+	// If no labels are found, bail.
+	if ( repoLabels.length === 0 ) {
+		debug( 'triage-issues: No labels found in the repository. Aborting OpenAI request.' );
+		return suggestions;
+	}
+
+	const prompt = `Here is the issue title. It is the most important part of the text you must analyse:
+${ title }
+
+Here is the issue body:
+${ body }
+
+Here are the existing labels and their descriptions:
+${ repoLabels.join( ', ' ) }
+
+Analyze the issue and suggest relevant labels. Rules:
+- Don't search for and add more labels if the existing labels match your initial assessment.
+- Use only existing labels provided.
+- Include 1 '[Feature Group]' label.
+- Include 1 to 3 '[Feature]' labels.
+- Briefly explain each label choice in 1 sentence.
+- Format your response as a JSON object with 'labels' and 'explanations' keys.
+
+Example response format:
+{
+    "labels": ["[Product] WordPress.com", "[Feature Group] User Interaction & Engagement", "[Feature] Comments"],
```

Comment:
Not sure if this might confuse the AI when it was told not to include "[Product]" labels and when no explanation for the label is included.
```suggestion
    "labels": ["[Feature Group] User Interaction & Engagement", "[Feature] Comments"],
```